src/app/learn/reference/naming-and-placement/index.html

```diff
@@ -38,6 +38,11 @@
         <sky-code>src/app/lib</sky-code> folder — This folder is the recommended location for files to exclude from code coverage. SKY UX's code coverage instrumentation ignores all files in the <sky-code>lib</sky-code> folder.
       </p>
     </li>
+    <li>
+      <p>
+        SCSS file — To add custom styles, you create the SCSS file and add <sky-code>@import '~@skyux/theme/scss/mixins';</sky-code> to reference variables or mixins. For more information, see the <a href="http://sass-lang.com/">Sass website</a>.
```

Comment:
Where do users create this file? I think we should say where to create the file. My original feedback suggested moving this item here because I thought it goes in the `src/app` folder. I think we should check with a developer to confirm the location and specify it in the description.

Also, where do consumers add `@import '~@skyux/theme/scss/mixins';`? Is it in this SCSS file? I assume so from how this is written, but I want to make sure.

Also, The last sentence seems entirely out of context. I'd add `SCSS is a superset of CSS3 syntax.` befroe the last sentence to provide context.